sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -674,6 +680,8 @@ public void post(@NonNull Runnable runnable, Runnable disabledRunnable) {
                 handlerAppCenterOperation(runnable, disabledRunnable);
             }
         };
+        mApplicationLifecycleListener = new ApplicationLifecycleListener(mHandler);
+        mApplication.registerActivityLifecycleCallbacks(mApplicationLifecycleListener);
```

Comment:
shouldn't that register be at same place as the existing register?

Reply:
I see there is one already at the right place, thus do we really need this one?

Reply:
We put `mApplication.registerActivityLifecycleCallbacks(mApplicationLifecycleListener);`
 in the `startService` method at first. But it caused events duplication:
`startService` is called multiple times and we registered to activity lifecycle callbacks multiple times. 

Of course, it could be fixed with a flag, but this approach seems simpler.

Reply:
Why do we also need it at https://github.com/microsoft/appcenter-sdk-android/pull/1385/files#diff-918508e799dc541fd29d897160e9b0a5R935 then?

Reply:
@guperrot that line subscribes services to application (not activity) lifecycle events.